tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_create_vol_component_exist_name.py

```diff
@@ -47,8 +55,45 @@ def run(ceph_cluster, **kw):
         if output1 == 0 or output2 == 0 or output3 == 0:
             return 1
         return 0
-
     except Exception as e:
-        log.info(e)
-        log.info(traceback.format_exc())
+        log.error(e)
+        log.error(traceback.format_exc())
         return 1
+
+
+def create_fs(client, vol_name, build, validate=True, **kwargs):
+    """
+    This Function creates the cephfs volume with vol_name given
+    It validates the creation operation by default.
+    It supports below optional arguments also
+    Args:
+        client:
+        vol_name:
+        build:
+        validate:
+        **kwargs:
+            check_ec = True
+    Returns:
+    """
+    if build == "4":
+        fs_cmd = f"ceph fs create {vol_name}"
+        cmd_out, cmd_rc = client.exec_command(
+            sudo=True, cmd=fs_cmd, check_ec=kwargs.get("check_ec", True)
+        )
+        if validate:
+            out, rc = client.exec_command(sudo=True, cmd="ceph fs ls --format json")
+            volname_ls = json.loads(out.read().decode())
+            if vol_name not in [i["name"] for i in volname_ls]:
+                raise CommandFailed(f"Creation of filesystem: {vol_name} failed")
+        return cmd_out, cmd_rc
+    if build == "5":
+        fs_cmd = f"ceph fs volume create {vol_name}"
+        cmd_out, cmd_rc = client.exec_command(
+            sudo=True, cmd=fs_cmd, check_ec=kwargs.get("check_ec", True)
+        )
+        if validate:
+            out, rc = client.exec_command(sudo=True, cmd="ceph fs ls --format json")
+            volname_ls = json.loads(out.read().decode())
```

Comment:
Can see this works as they made changes to output for exec_command.
We don't need read().decode()

Reply:
@AmarnatReddy Can you give me more details regarding to the question?

Reply:
@AmarnatReddy I have applied those changes. thanks